Description: Update Makefile for building with Debian
Author: Hiroyuki Yamamoto <yama1066@gmail.com>
Last-Update: 2019-01-29, Markus Frosch <lazyfrosch@debian.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -8,6 +8,14 @@
 
 CC=gcc
 CFLAGS=-O
+LDFLAGS=-lncurses
+
+all: sl
 
 sl: sl.c sl.h
-	$(CC) $(CFLAGS) -o sl sl.c -lncurses
+	$(CC) $(CFLAGS) -o sl sl.c $(LDFLAGS)
+
+clean:
+	rm -f sl
+
+distclean: clean
